website/src/commands/append.md

```diff
@@ -1,6 +1,6 @@
 # git town append
 
-> _git town append [--prototype] &lt;branch-name&gt;_
+> `git town append [--prototype] <branch-name>`
```

Comment:
Formatting this as code looks great, more like the technical documentation that this is. Thanks for cleaning this up! The italics were just an experiment, and I probably got a bit too creative there :laughing: